build.fsx

```diff
@@ -19,35 +27,58 @@ let sublimePath () =
     [  Environment.GetEnvironmentVariable("SUBLIME_TEXT_DATA")
        Environment.ExpandEnvironmentVariables(@"%APPDATA%\Sublime Text 3") ]
 
-  let isWindows = (Path.DirectorySeparatorChar = '\\')
   let searchPaths = if isWindows then WindowsPaths else UnixPaths
   let directories =
      searchPaths
      |> List.filter Directory.Exists
 
   match directories.Length with
-     | 0 ->
-         trace "No Sublime text 3 installation found"
+  | 0 -> trace "No Sublime text 3 installation found"
          exit 1
-     | _ ->
-         directories.Head
+  | _ -> directories.Head
+
+let getSublimeStartArgs () = 
+    if not isWindows then
+        let isRunning = getProcessesByName("Sublime Text") |> Seq.length > 0
```

Comment:
Not sure if the F# way is to separate parens from func names in arg lists. I've seen both, I think.